### DISCUSSION TITLE
Video Freezes when i click a button on the remote page

### DISCUSSION THREAD

**Discussion by David1946** [GerryDazoo/Slinger#73] [Q&A] [answered]
Now that I have got the Slinbox-Server working I've been trying out the remote control aspect and I have found that whenever I click on e.g the CH+ button the video freezes but the audio changes to the new channel. So the button is working to some extent but all I have is a still picture from the previous channel and the audio from the new channel. As this has not been raised before in the discussions I presume that it is something that I am doing wrong. Any advice would be much appreciated.

**Comment by GerryDazoo** [comment]
Not necessarily you. I'm guessing it's a problem with your media player. What are you using?

**Reply by David1946** [reply]
Thanks I use VLC.

**Comment by f520k510** [comment, marked as answer]
I suggest use mpv player for mac.  I just noticed with vlc I had the same issue, too.  I would always disconnect and re-connect the stream when that happened.  With mpv player I don't have this issue.

**Reply by David1946** [reply]
Okay Thanks I'll try that.

**Reply by David1946** [reply]
That's maybe beyond my level. I wouldn't know where to start with it.
